### PR TITLE
Singulo beacons and powersinks have more accurate feedback messages.

### PR DESCRIPTION
**Pull request by harmonyn**
To stop newbies and me from ahelping OMG THIS IS OVER AN EXPOSED CABLE WHY DOESN'T IT WORK??? every time they try to use one of these.

### CODE REVIEW

**Comment by Cheridan** [comment]
Thank you, Syndicate Tech Support

File: config/admins.txt [hunk 1]
```diff
@@ -7,7 +7,7 @@
 # NOTE: if the rank-name cannot be found in admin_ranks.txt, they will not be adminned! ~Carn #
 # NOTE: syntax was changed to allow hyphenation of ranknames, since spaces are stripped.      #
 ###############################################################################################
-Optimumtact = Host
+Optimumtact = Host\
```
Comment (Cheridan) [inline config/admins.txt]: Eh?

Comment (bgobandit) [inline config/admins.txt]: I've asked like 100 times in irc about how to make this stop autopushing

**Comment by ExcessiveUseOfCobblestone** [comment]
I'm pretty sure deadchat got a giggle from watching you trying to setup a beacon during that round.

File: config/admins.txt (hunk 1 above)
Comment (optimumtact) [inline config/admins.txt]: The question is why did you add a \?

anyway edit your .gitignore file

add config/admins.txt under *.backup.

you probably then want to add .gitignore under that so you don't push that change either.

**Comment by WJohn** [comment]
:+1: 
It should probably say "wire knot" instead of "wire node" but I guess that works already.
